pkg/sdk/grants.go

```diff
@@ -223,8 +223,13 @@ func (row grantRow) convert() *Grant {
 	var granteeName AccountObjectIdentifier
 	if grantedTo == ObjectTypeShare {
 		parts := strings.Split(row.GranteeName, ".")
-		name := strings.Join(parts[1:], ".")
-		granteeName = NewAccountObjectIdentifier(name)
+		switch {
+		case len(parts) == 1:
+			granteeName = NewAccountObjectIdentifier(parts[0])
+		case len(parts) > 1:
```

Comment:
let's add logic explicitely for 2 and add a default (but here I think we should discuss if we have any default for such situation - like using last part always?) or should be change the method to return error).